.doks/doks-cluster/locals.tf

```diff
@@ -0,0 +1,281 @@
+######################
+# values_dbs_dev.yaml
+######################
+
+locals{
+  template_values_dbs_dev = <<EOF
+global:
+  defaultdb:
+    create: false
+    postgresqlUsername: ${digitalocean_database_cluster.postgres-dev.user}
+    postgresqlDatabase: postgres-dev-pool-defaultdb
+    postgresqlPassword: ${digitalocean_database_cluster.postgres-dev.password}
+    host: ${digitalocean_database_connection_pool.postgres-dev-pool-defaultdb.private_host}
+    service:
+      port: ${digitalocean_database_connection_pool.postgres-dev-pool-defaultdb.port}
+  analysisdb:
+    create: false
+    postgresqlUsername: ${digitalocean_database_cluster.postgres-dev.user}
+    postgresqlDatabase: postgres-dev-pool-analysis
+    postgresqlPassword: ${digitalocean_database_cluster.postgres-dev.password}
+    host: ${digitalocean_database_connection_pool.postgres-dev-pool["analysis"].private_host}
+    service:
+      port: ${digitalocean_database_connection_pool.postgres-dev-pool["analysis"].port}
+  resultsdb:
+    create: false
+    postgresqlUsername: ${digitalocean_database_cluster.postgres-dev.user}
+    postgresqlDatabase: postgres-dev-pool-results
+    postgresqlPassword: ${digitalocean_database_cluster.postgres-dev.password}
+    host: ${digitalocean_database_connection_pool.postgres-dev-pool["results"].private_host}
+    service:
+      port: ${digitalocean_database_connection_pool.postgres-dev-pool["results"].port}
+  metricsdb:
+    create: false
+    postgresqlUsername: ${digitalocean_database_cluster.postgres-dev.user}
+    postgresqlDatabase: postgres-dev-pool-metrics
+    postgresqlPassword: ${digitalocean_database_cluster.postgres-dev.password}
+    host: ${digitalocean_database_connection_pool.postgres-dev-pool["metrics"].private_host}
+    service:
+      port: ${digitalocean_database_connection_pool.postgres-dev-pool["metrics"].port}
+  filestoredb:
+    create: false
+    postgresqlUsername: ${digitalocean_database_cluster.postgres-dev.user}
+    postgresqlDatabase: postgres-dev-pool-filestore
+    postgresqlPassword: ${digitalocean_database_cluster.postgres-dev.password}
+    host: ${digitalocean_database_connection_pool.postgres-dev-pool["filestore"].private_host}
+    service:
+      port: ${digitalocean_database_connection_pool.postgres-dev-pool["filestore"].port}
+  jobsdb:
+    create: false
+    postgresqlUsername: ${digitalocean_database_cluster.postgres-dev.user}
+    postgresqlDatabase: postgres-dev-pool-jobs
+    postgresqlPassword: ${digitalocean_database_cluster.postgres-dev.password}
+    host: ${digitalocean_database_connection_pool.postgres-dev-pool["jobs"].private_host}
+    service:
+      port: ${digitalocean_database_connection_pool.postgres-dev-pool["jobs"].port}
+activities:
+  activitiesdb:
+    create: false
+    postgresqlUsername: ${digitalocean_database_cluster.postgres-dev.user}
+    postgresqlDatabase: "activities?sslmode=require"
```

Comment:
This db was failing to receive connections. The Activities pod was entering an `Error` state because the connection was non-ssl. Then I tried (with the amazing help of @h314to kudos!) to connect to the connection pool with SSL and there were still other errors occurring. The only way in which we managed to establish a connection with SSL enabled was to not use the connection pool, and to pass the ssl mode as part of the db name, which will get appended to the connection string that goes on the component's configmap.